mlflow/legacy_databricks_cli/configure/provider.py

```diff
@@ -276,8 +287,8 @@ def get_config(self):
 class ProfileConfigProvider(DatabricksConfigProvider):
     """Loads from the databrickscfg file."""
 
-    def __init__(self, profile=DEFAULT_SECTION):
-        self.profile = profile
+    def __init__(self, profile=None):
+        self.profile = profile or DEFAULT_SECTION
```

Comment:
why do we need this change?

Reply:
some code has small refactoring in `databricks_utils` so that in some cases `ProfileConfigProvider(profile=None)` is invoked, in this case, we need `ProfileConfigProvider` to recognize the profile to be `DEFAULT`

Reply:
got it